frontend/src/NavBar.js

```diff
@@ -10,7 +10,7 @@ class NavBar extends React.Component {
 
   render() {
     return (
-        <Navbar fixed="top" collapseOnSelect expand="lg" bg="dark" variant="dark">
+        <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
             <Navbar.Brand href="#home">Microapp</Navbar.Brand>
```

Comment:
It's a nit but now that I think of it, we can actually rename it to Hyperlocal Spot or something similar

Reply:
SGTM, thanks - I'll make that change in one of the upcoming PRs